dist/page-grid/page-grid.css

```diff
@@ -33,17 +33,25 @@
   position: relative;
   width: 100%;
 }
-.page-grid-container .page-grid-container {
-  max-width: 100%;
-  padding: 0;
-  width: 100%;
-}
 .page-grid {
   column-gap: var(--page-grid-column-gaps);
   display: grid;
   grid-template-columns: repeat(var(--page-grid-number-cols), var(--page-grid-column-widths));
   row-gap: var(--page-grid-row-gaps);
 }
+.subgrid {
```

Comment:
Should we name this `page-grid__subgrid`?

Reply:
Hmm...maybe. I suppose since `.subgrid` can't exist on its own, it makes sense to make it a true child. Will changing it make this a breaking change though? Is that fine?

Reply:
We could use a grouped selector and phase out `.subgrid` on next major version.

Reply:
Done.